src/main/java/DukeLevel.java.rej

```diff
@@ -0,0 +1,151 @@
+diff a/src/main/java/DukeLevel.java b/src/main/java/DukeLevel.java	(rejected hunks)
+@@ -1,3 +1,4 @@
++import java.io.*;
```

Comment:
Using the wildcard import is discouraged in this module. Maybe you can consider importing the relevant features individually?